docs/monitoring/infrastructure.md

```diff
@@ -1,35 +1,59 @@
 
+![Infra App screen](../images/monitoring/infra-app.png)
+
 Each Infra App is typically used to gather data from one environment. For example, one Infra App for production environment, one for test environment and one for development environment.
 
-Sematext Cloud will automatically create one Infra App named "Default Infra App" the first time you might need it. You can also create Infra App on your own under Integrations -> Overview.
+![Infra Env](../images/monitoring/infra-env.png)
+
+Sematext Cloud will automatically create one Infra App named "Default Infra App" the first time you might need it. 
+
+You can also create Infra App on your own under Apps > New App > Monitoring. Check below for more information.
 
 ## How it works
 
 A Golang-based [Sematext Agent](../agents/sematext-agent) will be installed on your machine. This Agent sends all infrastructure data to an Infra App whose token is configured during the Agent's installation.
 
 The Infra App also includes data about your containerized environments. Installation instructions for container monitoring include an Infra App token. That way when installing Sematext Agent, an Infra App will be installed as well.
 
+## Create an Infra App
+
+Creating a new Infra App is the same process as with any App in Sematext. In the Apps screen you choose New App > Monitoring > Infra App. Give it a name and click 'Create App'.
+
+![New Infra App](../images/monitoring/new-infra-app-env.gif)
+
+During the Infra App creation process, you will get a screen displaying Sematext Agent installation instructions for various supported environments. You will need to follow these instructions to use of the Infra App.
+
+## Infra App Metrics
+
+Metrics will start shipping to your new Infra App in a few moments. You can see metrics from all your servers, Kubernetes clusters, containers, etc.
+
+![Infra App metrics](../images/monitoring/infra-metrics.gif)
+
 ## Enabling Infra Monitoring
 
+Infra Monitoring is enabled by default as soon as you install Sematext Agent.
+
 If you've set up Sematext Monitoring at some point in the past, you can easily enable Infra Monitoring by:
 
-- [Upgrading](./spm-faq/#agent-updating) your `sematext-agent` to the latest version
+- [Upgrading](./spm-faq/#agent-updating) your Sematext Agent to the [latest version](https://sematext.com/docs/agents/sematext-agent/releasenotes/)
 - Setting up Infra App token on each of your machines by running the `setup-infra` command
 
-> **Note**: For Docker monitoring, we suggest uninstalling the existing deprecated Agent and going through the new setup steps for Docker Monitoring in Sematext Cloud.
+> **Note**: For Docker monitoring, we suggest uninstalling the existing deprecated Agent and going through the new setup steps for Docker monitoring in Sematext Cloud.
```

Comment:
Maybe we can remove this statement?  We added it a long time ago when we had some old agent version that we deprecated (a few years back).

Reply:
Yes